test/test_torch.py

```diff
@@ -7915,6 +7917,34 @@ def test_serialization(self):
                 f.seek(0)
                 c = torch.load(handle)
             self._test_serialization_assert(b, c)
+        # test non-ascii encoding of bytes arrays/strings
+        # The following bytes are produced by serializing
+        #   [u'żąąóżąż'.encode('utf-8'), torch.zeros(1, dtype=torch.float), 2]
```

Comment:
fbcode restrictions dont allow non-ascii characters in files, i think. i'll have to import and modify, can you just modify it here instead

Reply:
Sure. I'll fix it here.

Reply:
Fixed :)

Reply:
Interesting, it produces pretty much exactly the set of special characters for Polish 🤔